lib/pluggable.js

```diff
@@ -0,0 +1,115 @@
+// TODO: make DRY
+
+import * as most from "most";
+
+
+export const CONTINUE = Symbol.for("interlock.continue");
+
+
+function getContext (baseCxt, dependencies) {
```

Comment:
I should not have been doing this so late :/ my mistake. I was reading that bind as a modification to dep but it really just returns a new function so I stand corrected - there are no side effects.